src/components/Keypad/Keypad.spec.js

```diff
@@ -18,7 +18,19 @@ describe('Keypad', () => {
         );
     });
 
-    it('renders a <div />', () => {
-        expect(subject.find('div').length).toBe(1);
+    it('renders a div with a classname of number-container', () => {
+        expect(subject.find('.numbers-container').length).toBe(1);
     });
+
+    it('renders the keypad values', () => {
+        expect(subject.find('.numbers-container').text()).toEqual('012');
```

Comment:
```suggestion
        expect(subject.find('.numbers-container').text()).toBe('012');
```
I wouldn't block you on this, but I err on the side of being as strict in my comparison as possible. More info:
https://jestjs.io/docs/en/using-matchers#common-matchers